agent/agent.go

```diff
@@ -597,6 +600,9 @@ func (a *Agent) Start(ctx context.Context) error {
 	// Overwrite the configuration.
 	a.config = c
 
+	a.enableDebug = atomic.Bool{}
```

Comment:
I think the atomic default value is `atomic.Bool{}` so no need to initialize it here.